content/unexpected-reboots.md

```diff
@@ -0,0 +1,441 @@
+    warning-level:       none
+    energy:              69.54 Wh
+    energy-full:         70.1784 Wh
+    energy-rate:         25.6956 W
+    percentage:          99%
+    icon-name:          'battery-full-charged-symbolic'
+
+Daemon:
+  daemon-version:  0.99.11
+  on-battery:      no
+  lid-is-closed:   no
+  lid-is-present:  yes
+  critical-action: HybridSleep
+```
+
+Specific sections to pay attention to are `capacity`, `energy-full`, and `energy-full-design`. These indicate how well the battery is aging, and assessing it's own charging capabilities.
+
+```
+    energy-full:         70.1784 Wh
+    energy-full-design:  70.68 Wh
+```
+
+`energy-full` should never be higher than `energy-full-design`. If it is, that indicates a problem with the integrated chip on the battery, and may be why charging reports are incorrect, and why, by extension, the system is shutting off unexpectedly. 
+
+`capacity` is a good way to gauge the age and health of the battery. If capacity is 70% or lower, the battery is showing aging, and won't hold as much charge as it did when new. 
+
+>**NOTE:** This number will gradually decrease over time from ~100% to 0%. Having a capacity of 95% and steadily decreasing as the battery gets older is expected behavior. Sudden drops in capacity, however, can indicate hardware failure.
+
+#### AC Adapter
+
+There are four (4) main components that can be the cause of spontaneous shut-offs related to the AC Adapter:
+
+1. The cable from the wall outlet to the PSU/"brick".
+2. The PSU/"brick" and the DC cable to the laptop.
+3. DC-In Port on the laptop.
+4. Charging circuitry on the mainboard.
+
+Stress to the cabling, or short-circuits on components 1 or 2 can cause shut-offs. If the cable is moved while connected and the damaged part of the cable shifts enough, current into the laptop is interrupted. If the battery is removed, or no longer holding a charge, this interruption can cause the system to power down.
+
+Removing the battery, connecting the AC adapter and moving or rotating the cable components one section at a time can reveal which part of the cabling may need replaced.
+
+If moving the plug in the DC-In port on the laptop specifically causes the issue, it's more likely that the DC-In port needs replaced or re-soldered onto the board.
+
+Both the DC-In port and issues with the mainboard charging circuitry require systems to be brought in for repairs. To open a support ticket, visit: [this link](https://system76.com/my-account/support-tickets/new)
+
+## Desktop Specific Troubleshooting
+
+The main difference in troubleshooting desktop reboots vs. laptop reboots, is the battery, and greater disassembly options (i.e. more parts to troubleshoot). With a desktop computer (excepting the Meerkat), the power delivery system consists of the PSU, and the cable from the PSU to the wall. Unless a battery backup is added via an Uninterrupted Power Supply (UPS) an issue in the power delivery system will shut the system off.
```

Comment:
In the first sentence here, there shouldn't be a comma before "is" or after "battery," and you should probably say "the main differences are" since you're specifying two differences.

Reply:
Fixing.

Reply:
Fixed.